01_prepare_host.sh

```diff
@@ -43,6 +43,14 @@ if ! command -v kustomize 2>/dev/null ; then
     sudo mv kustomize /usr/local/bin/.
 fi
 
+if [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
```

Comment:
this variable needs to be exported and defaulted in lib/common.sh

Reply:
Should "minikube" be the default then?

Reply:
yes, Minikube should be default

Reply:
Fixed